pkgdb/Makefile

```diff
@@ -633,13 +633,8 @@ cc-check: $(TESTS:.cc=)
 	exit "$$_ec";
 
 #: Run all bats tests
-BATS_FILE ?= $(MAKEFILE_DIR)/tests
 bats-check: bin $(TEST_UTILS)
-	PKGDB="$(MAKEFILE_DIR)/bin/pkgdb"                        \
-	IS_SQLITE3="$(MAKEFILE_DIR)/tests/is_sqlite3"            \
-	SEARCH_PARAMS="$(MAKEFILE_DIR)/tests/search-params"      \
-	  bats --print-output-on-failure --verbose-run --timing  \
-	       "$(BATS_FILE)";
+	flox-pkgdb-tests
```

Comment:
non-blocking: this is fine as long as the underlying test script does not depend on other _built_ packages from `github:flox/flox`, and it respects environment variables passed in as `FOO=1 flox-pkgdb-tests`.